presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/RequestDispatcher.java

```diff
@@ -65,32 +69,39 @@ public Response forwardRequest(String scope, ApiRequester apiRequester, Collecti
 
         Collection<URI> uriCollection;
         try {
-            uriCollection = nodeId.isEmpty() ? nodeSet.getUris(apiScope) : nodeSet.getUrisByIds(nodeId);
+            uriCollection = nodeId.isEmpty()
+                    ? nodeSet.getUris(apiScope)
+                    : nodeSet.getUrisByIds(nodeId);
         }
         catch (IllegalArgumentException e) {
             LOGGER.error("Invalid or duplicate node ID");
-            return Response.status(BAD_REQUEST).entity("Invalid or duplicate node ID").build();
+            return Response.status(BAD_REQUEST)
+                    .entity("Invalid or duplicate node ID").build();
         }
 
         if (apiScope == ApiScope.COORDINATOR && uriCollection.size() != 1) {
             LOGGER.error("Number of coordinator is not 1");
-            return Response.status(INTERNAL_SERVER_ERROR).entity("Number of coordinator is not 1").build();
+            return Response.status(INTERNAL_SERVER_ERROR)
+                    .entity("Number of coordinator is not 1").build();
         }
 
         List<Response> responseList = new ArrayList<>();
         for (URI uri : uriCollection) {
             responseList.add(apiRequester.send(uri));
         }
+        List<ResponseWrapper> responses = responseList.stream()
+                .map(ResponseWrapper::wrapResponse)
+                .collect(Collectors.toList());
```

Comment:
How about `ImmutableList.toImmutableList()` from Guava? I thought you have Guava available in this module.

Anyway, given your next PR is open, we can skip that for now.